docs/data/external-oasis-repos.md

```diff
@@ -63,94 +75,186 @@ root@host # chkconfig httpd on
-2.  When it is time to put the repository back into production, the GOC representative runs `unblank_osg_repository` on oasis-replica and gives it the full name of the repository again. This will find the directory with the old timestamp and put it back into service. This step also attempts to ssh to the oasis machine.
+If the repository ends in `.opensciencegrid.org`, the VO may ask for it to be replicated outside the US.  The
+VO should open a GGUS ticket following EGI's [PROC20](https://wiki.egi.eu/wiki/PROC20).
 
-Procedure to change the URL of an external repository
------------------------------------------------------
+Change the URL of an external repository
+----------------------------------------
 
-1.  The repository service administrator opens a GOC ticket with the value of the new URL.
-2.  The GOC representative then changes the registered value in OIM for the VO in OASIS Repo URLs. The GOC stratum 1 will then be updated within an hour.
+If necessary, it is possible to change the URL of the repository server:
+
+1.  The repository administrator opens a support ticket with the value of the new URL.
+2.  The OSG representative then changes the registered value in OIM for the VO in OASIS Repo URLs.
+
+The GOC Stratum-1 will then be updated within an hour.
+
+Remove an external repository
+-----------------------------
+
+1.  If the repository has been replicated outside of the U.S., the repository service administrator should open a GGUS
+    ticket asking that the replication be removed from EGI Stratum-1s. Wait until this is done before proceeding 
+2.  The repository administrator opens an OSG support ticket asking to shut down the repository, giving the repository
+    name (e.g., %RED%example.opensciencegrid.org%ENDCOLOR%) and the corresponding VO.
+3.  After validating the ticket submitter is authorized by the VO's OASIS manager, the OSG representative next deletes
+    the registered value for %RED%example.opensciencegrid.org%ENDCOLOR% in OIM for the VO in OASIS Repo URLs. 
+4.  The OSG representative adds the FNAL and BNL Stratum-1 operators to the ticket to asks them to remove the
+    repository.  Wait for the Stratum-1 operators to finish before proceeding.
+5.  The OSG representative then runs `cvmfs_server rmfs -f %RED%example.opensciencegrid.org%ENDCOLOR%`
+    and `rm -r /oasissrv/cvmfs/%RED%example.opensciencegrid.org%ENDCOLOR%` on `oasis-replica-itb` and `oasis-replica`
+6.  The OSG representative does `rm -r /srv/cvmfs/%RED%example.opensciencegrid.org%ENDCOLOR%` on `oasis-itb` and
+    `oasis`.
```

Comment:
Ditto for this section:

> 1. If the repo has been replicated outside of the U.S. Open a GGUS ticket and wait until
> 1. Open a OSG support ticket (\<link\>) with the following information...

The rest should be shipped off to the relevant technology/operations doc. It's also not clear if the ticket in step 1 has to be resolved or just submitted before starting on the rest of the steps.

Reply:
Fixed the minor issue with (1), but will wait on the others for the same reason as above.